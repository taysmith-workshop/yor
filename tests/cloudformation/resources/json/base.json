{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        },
        "Tags": [
          {
            "Key": "old_tag",
            "Value": "old_value"
          },
          {
            "Key": "yor_trace",
            "Value": "0d37f3ef-ae10-487f-8a5c-66485b24a3fb"
          }
        ]
      },
      "DeletionPolicy": "Retain"
    }
  }
}