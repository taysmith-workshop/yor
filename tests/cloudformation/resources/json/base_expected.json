{
  "AWSTemplateFormatVersion": "2010-09-09",
  "Resources": {
    "S3Bucket": {
      "Type": "AWS::S3::Bucket",
      "Properties": {
        "AccessControl": "PublicRead",
        "WebsiteConfiguration": {
          "IndexDocument": "index.html",
          "ErrorDocument": "error.html"
        },
        "Tags": [
          {
            "Key": "old_tag",
            "Value": "old_value"
          },
          {
            "Key": "new_tag",
            "Value": "new_value"
          },
          {
            "Key": "another_tag",
            "Value": "another_val"
          },
          {
            "Key": "yor_trace",
            "Value": "80243750-670d-439a-8138-74e1228c3f6a"
          }
        ]
      },
      "DeletionPolicy": "Retain"
    }
  }
}